src/components/v2/appDetails/sourceInfo/environmentStatus/AppStatusDetailModal.tsx

```diff
@@ -35,7 +35,7 @@ function AppStatusDetailModal({ close, appStreamData, showAppStatusMessage }: {
     const [showSeeMore, setShowSeeMore] = useState(true)
 
     useEffect(() => {
-        const stats = appStreamData?.result?.application?.status.operationState.syncResult.resources.reduce(
+        const stats = appStreamData?.result?.application?.status.operationState.syncResult?.resources.reduce(
```

Comment:
Are we sure that status and status.operationState will always be there?

Reply:
Changes are done